### ISSUE TITLE
How to train weights in confidence branch?

### ISSUE THREAD

**Issue by jevonswang** [karfly/learnable-triangulation-pytorch#60]
Hi,
     I notice that the model get prediction confidence by appending a GlobalAveragePoolingHead on feature. However, the loss is only calculated by keypoints_3d_pred. Without enough supervision signal, how can the GlobalAveragePoolingHead update its weights? Looking forward to your reply.

Best,
Jevons

**Comment by jevonswang** [comment]
Oh, I know. Confidences are used in triangulation phase.
